src/app/credExplorer/pagerankTable/TableRow.test.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+
+import React from "react";
+import {shallow} from "enzyme";
+import {TableRow} from "./TableRow";
+
+import {COLUMNS} from "./sharedTestUtils";
+require("../../testUtil").configureEnzyme();
+
+describe("app/credExplorer/pagerankTable/TableRow", () => {
+  function example(depth: number = 3) {
+    return shallow(
+      <TableRow
+        depth={depth}
+        description={<span testDescription={true} />}
+        connectionProportion={0.5}
+        score={133.7}
+        children={<div testChildren={true} />}
+      />
+    );
+  }
+  it("has depth-based color", () => {
+    for (const depth of [0, 1, 2]) {
+      const el = example(depth);
+      const style = el.find("tr").props().style;
+      expect({depth, style}).toMatchSnapshot();
+    }
+  });
+  it("has depth-based indentation", () => {
+    for (const depth of [0, 1, 2]) {
+      const el = example(depth);
+      const style = el
+        .find("td")
+        .at(0)
+        .props().style;
+      expect({depth, style}).toMatchSnapshot();
```

Comment:
What is this snapshot supposed to tell you? The old snapshot was useful:
it said

```js
exports[`[SNIP] ConnectionRow has proper depth-based styling 1`] = `
Object {
  "buttonStyle": Object {
    "marginLeft": 30,
    "marginRight": 5,
  },
  "trStyle": Object {
    "backgroundColor": "rgba(0,143.4375,0,0.18999999999999995)",
  },
}
`;
```

but the new snapshot only says

```js
exports[`app/credExplorer/pagerankTable/TableRow has depth-based indentation 1`] = `
Object {
  "depth": 0,
  "style": Object {
    "alignItems": "flex-start",
    "display": "flex",
  },
}
`;
```

Reply:
Oops, the snapshot was checking the wrong element's style. Fixed. (I separated them out because I was anticipating that a future change will cause the indentation and color to vary independently. The tests will wind up in a clean state after I change how depth styling works.)